governance/SIG Security Charter.md

```diff
@@ -1,97 +1,73 @@
 # SIG Security Charter
 
-This charter adheres to the Roles and Organization Management specified in <sig-governance>.
- Team information may be found in the <readme.md>
-
 ## Overview of SIG
-A central knowledge advisory group that provides resources and information security compliant, resolution and XYZ.
-
-All about security of systems and infrastructure resources. Security patching, resources and infra up to date.
-Provide the basic services of APPSEC related to this project.
-
-Two concise lines explaining what this SIG does with bullet points of the major responsibilities
-
-- Responsibility
+SIG Security is a special interest group for O3DE code security that works as a central knowledge advisory group and  provides resources, consultation, reporting mechanisms  and information for security compliance, and security related issue resolution. This SIG also helps in keeping security patching up to date, and provide the basic services related to  APPSEC (Application Security) of O3DE.
 
 ## Goals
 
-- Major goals that SIG seeks to generally achieve
+- To be set after rest of the charter is reviewed.
 
 ## Scope
-Involve gathering information about security flaws. 
-Identify potential security risks/gaps in softare.
-Define best practices for ensuring secure code
-
-- Generalized overall scope of work
+-  Involves gathering information about security flaws. 
```

Comment:
To match stance of other Scope items:
`-  Gather information about...`

Reply:
```suggestion
-  Gather information about security flaws. 
```